### PR TITLE
Future Work Extension [Ezgi]

### PR DESCRIPTION
**Pull request by ezgimez**
future work extension

### CODE REVIEW

File: 05_iigo/index.tex [hunk 1]
```diff
@@ -467,16 +467,28 @@ \section{IIGO Session order}
...
-            \item Revoking an island's eligibility to propose a rule/motion.
+            \item Revoking an island's eligibility to propose any rule or a specific rule.
         \end{itemize}
-    \item \textbf{Immutable rules}: Within the scope of the coursework, a subset of rules could have been categorised as immutable. This means that to change such immutable rules, the islands first need to vote to change their status to be \emph{mutable}, and consequently, hold another vote to change these mutable rules.
+\textbf{Immutable rules.} Within the scope of the coursework, a subset of rules could have been categorised as immutable. This means that to change such immutable rules, the islands first need to vote to change their status to be \emph{mutable}, and consequently, hold another vote to change these mutable rules. An example of such an immutable rule could have been "An island in the archipelago cannot hold multiple IIGO roles (i.e. the President, Speaker, and Judge) in the same turn if at least two other islands are still alive.". This immutable rule would allow islands to avoid the tyranny that might arise from the lack of separation of power in any turn.\\
```
Comment (Spruudis) [inline 05_iigo/index.tex:897]: "An island in the archipelago cannot hold multiple IIGO roles (i.e. the President, Speaker, and Judge) in the same turn if at least two other islands are still alive." is an extension in of itself and should be grouped together with other rules we could introduce

File: 05_iigo/index.tex [hunk 2]
```diff
@@ -467,16 +467,28 @@ \section{IIGO Session order}
...
     %\item \textbf{Adding rules to the proposal list: } 
-\end{itemize}
+\textbf{Scope of IIGO.} Within the scope of the coursework, additional rules could be defined to audit levels of interaction at the IITO, IIFO and foraging. \\
```
Comment (Spruudis) [inline 05_iigo/index.tex:899]: Isn't this section about the stuff that is out of scope for this coursework?

File: 05_iigo/index.tex [hunk 3]
```diff
@@ -467,16 +467,28 @@ \section{IIGO Session order}
...
+    Suggested additional rules include:
+        \begin{itemize}
+            \item During an IITO session, each island is not \emph{permitted} to give gifts to a sanctioned island.
```
Comment (Spruudis) [inline 05_iigo/index.tex:902]: Maybe move to sanction expansion?

Comment (Spruudis) [inline 05_iigo/index.tex:902]:
```suggestion
            \item During an IITO session, each island is not \emph{permitted} to give gifts to a sanctioned island.
            \item Each island is \emph{obliged} to send the gift amount it agreed to send during the IIGO session. 
```

File: 05_iigo/index.tex (hunk 2 above)
Comment (ezgimez) [reply 05_iigo/index.tex:899]: no.. for ''future work''

File: 05_iigo/index.tex (hunk 3 above)
Comment (ezgimez) [reply 05_iigo/index.tex:902]: there is no sanction expansion

Comment (ezgimez) [reply 05_iigo/index.tex:902]: I don't think we should create one

File: 05_iigo/index.tex (hunk 1 above)
Comment (ezgimez) [reply 05_iigo/index.tex:897]: I didn't know you had in mind a section called ''rules we could introduce'' in mind

Comment (ezgimez) [reply 05_iigo/index.tex:897]: I can keep it like this, you can PR on  top of this?

File: 05_iigo/index.tex (hunk 3 above)
Comment (ezgimez) [reply 05_iigo/index.tex:902]: again, if you wanna introduce more sections in the future work --
we can merge this PR --> AND THEN you can revise sections or smth...
I cannot possibly know how many sections you want for future work if you cannot tell me

Comment (ezgimez) [reply 05_iigo/index.tex:902]: no. IITO is a jungle. 
there are no 'obligations' in IITO

Comment (ezgimez) [reply 05_iigo/index.tex:902]: there is not a _constitution_ in IITO which determnies the level of interactions there.

if you wanna use 'obligation' within IITO context, we should write it more clearly, not in just a **single** full sentence like this one

File: 05_iigo/index.tex (hunk 2 above)
Comment (ezgimez) [reply 05_iigo/index.tex:899]: oh ok

Comment (ezgimez) [reply 05_iigo/index.tex:899]: SCOPE OF IIGO --> meaning what IIGO can audit/punish/sanction or NOT

File: 05_iigo/index.tex (hunk 3 above)
Comment (Spruudis) [reply 05_iigo/index.tex:902]: I don't see why this could not be a rule agent can vote in and to as they please

Comment (ezgimez) [reply 05_iigo/index.tex:902]: oh ok. you are right. :)
but than this should not be included here , but at a different section called
"possible additional rules" or smth.
wouldn't this be better?

File: 05_iigo/index.tex [hunk 4]
```diff
@@ -467,16 +467,28 @@ \section{IIGO Session order}
...
+             \item During an IIFO session, a sanctioned island is not \emph{permitted} to learn and share any predictions regarding long-term collective risk dilemma (i.e. upcoming disasters) and short-term collective risk dilemma (i.e. returns from foraging)
```
Comment (preetl) [inline 05_iigo/index.tex]:
```suggestion
             \item During an IIFO session, a sanctioned island is not \emph{permitted} to learn and share any predictions regarding the long-term collective risk dilemma (i.e. upcoming disasters) and the short-term collective risk dilemma (i.e. returns from foraging)
```
